tests/Gateway.spec.ts

```diff
@@ -488,14 +469,61 @@ describe('Gateway', () => {
         // Try to trigger some TSS command
         // It should fail because external_message is not implemented anymore! :troll:
         try {
-            await gateway.sendEnableDeposits(tssWallet, true);
+            await gateway.sendEnableDeposits(deployer.getSender(), true);
         } catch (e: any) {
             // https://docs.ton.org/learn/tvm-instructions/tvm-exit-codes
             const exitCode = e?.exitCode as number;
             expect(exitCode).toEqual(11);
         }
     });
 
```

Comment:
We don't have unit test for update code and update TSS?

Reply:
wdym? It's there as 
- `it('should update tss address')`
- `it('should update the code')`

Reply:
Yeah got confused as it wasn't in the diff